opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -196,6 +204,36 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    public static class AsyncTimeoutServlet extends HttpServlet {
+
+        @Override
+        public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+            throws ServletException, IOException {
+
+            // avoid retries on timeout
+            if (request.getAttribute("timedOut") != null) {
+                return;
```

Comment:
@jpkrohling @objectiser if I don't do this the servlet is called multiple times. Any idea why?

Reply:
no idea - but seems strange, it is an auto retry mechanism?

Reply:
I don't know whether it is a jetty thing or servlet spec